qa/replace-by-fee/README.md

```diff
@@ -0,0 +1,13 @@
+Replace-by-fee regression tests
+===============================
+
+First get version v0.5.0 of the python-bitcoinlib library. In this directory
+run:
+
+    git clone -n https://github.com/petertodd/python-bitcoinlib
```

Comment:
I don't like hard-coding a commit ID here. It will go out of date very soon. If this requires v0.5.x, Can we do something like:

    git clone --branch 0.5 https://github.com/petertodd/python-bitcoinlib